pkg/dispatcher/dispatcher.go

```diff
@@ -55,13 +56,17 @@ func (d *Dispatcher) ListenUeEvents(ueEventChannel <-chan Event) {
 }
 
 // RegisterUeListener :
-func (d *Dispatcher) RegisterUeListener(subscriber string) (chan Event, error) {
+func (d *Dispatcher) RegisterUeListener(subscriber string, buffLength ...int) (chan Event, error) {
 	d.nbiUeListenersLock.Lock()
 	defer d.nbiUeListenersLock.Unlock()
+	var num int
+	if buffLength != nil {
+		num = buffLength[0]
+	}
 	if _, ok := d.nbiUeListeners[subscriber]; ok {
 		return nil, fmt.Errorf("NBI UE %s is already registered", subscriber)
 	}
-	channel := make(chan Event)
+	channel := make(chan Event, num)
```

Comment:
Is there any reason why this channel became buffered channel? Just curious.

Reply:
The Ue changes are buffered (in the Go channel) until they are dequeued periodically based on the configured L2MeasConfig timer interval.

Reply:
Oh, I got it.